google/cloud/BUILD.bazel

```diff
@@ -12,7 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-package(default_visibility = ["//visibility:public"])
+# TODO(#3701) Change the default_visibility to "//visibility:private", and
+# widen visibility as needed with per-target visibility.
+package(
```

Comment:
This file happens to have some libraries that have (AFAIK) no corresponding top-level target: `google_cloud_cpp_common` and `google_cloud_cpp_grpc_utils`.

Maybe for them we should decide if we are going to make these libraries public (I think we should).  And I think the deprecation period may need to be longer?